tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -23,25 +25,30 @@ class System(Base):
     _software_home_button_locator = (By.ID, 'software-home-button')
 
     def wait_for_status_bar_displayed(self):
-        self.wait_for_element_displayed(*self._status_bar_locator)
+        Wait(self.marionette).until(expected.element_displayed(*self._status_bar_locator))
 
     def wait_for_notification_toaster_displayed(self, timeout=10, message=None):
-        self.wait_for_element_displayed(*self._notification_toaster_locator)
+        Wait(self.marionette, timeout).until(
+            expected.element_displayed(*self._notification_toaster_locator), message=message)
```

Comment:
You still have `message=message` where just `message` would do, but it's just a nit, and there's no need to change it.